Hello_Console_stud/Hello_Console_stud/Program.cs

```diff
@@ -0,0 +1,229 @@
+            Console.Beep(264, 125);
+            Thread.Sleep(125);
+            Console.Beep(297, 500);
+            Thread.Sleep(125);
+            Console.Beep(264, 500);
+            Thread.Sleep(125);
+            Console.Beep(352, 500);
+            Thread.Sleep(125);
+            Console.Beep(330, 1000);
+            Thread.Sleep(250);
+            Console.Beep(264, 125);
+            Thread.Sleep(250);
+            Console.Beep(264, 125);
+            Thread.Sleep(125);
+            Console.Beep(297, 500);
+            Thread.Sleep(125);
+            Console.Beep(264, 500);
+            Thread.Sleep(125);
+            Console.Beep(396, 500);
+            Thread.Sleep(125);
+            Console.Beep(352, 1000);
+            Thread.Sleep(250);
+            Console.Beep(264, 125);
+            Thread.Sleep(250);
+            Console.Beep(264, 125);
+            Thread.Sleep(125);
+            Console.Beep(2642, 500);
+            Thread.Sleep(125);
+            Console.Beep(440, 500);
+            Thread.Sleep(125);
+            Console.Beep(352, 250);
+            Thread.Sleep(125);
+            Console.Beep(352, 125);
+            Thread.Sleep(125);
+            Console.Beep(330, 500);
+            Thread.Sleep(125);
+            Console.Beep(297, 1000);
+            Thread.Sleep(250);
+            Console.Beep(466, 125);
+            Thread.Sleep(250);
+            Console.Beep(466, 125);
+            Thread.Sleep(125);
+            Console.Beep(440, 500);
+            Thread.Sleep(125);
+            Console.Beep(352, 500);
+            Thread.Sleep(125);
+            Console.Beep(396, 500);
+            Thread.Sleep(125);
+            Console.Beep(352, 1000);
+        }
+        #endregion
+
+        #region Morse
+        static void Morse_code()
+        {
+            //Create string variable for 'sos'      
+            var sos = "sos";
+            //Use string array for Morse code
+            string[,] Dictionary_arr = new string [,] { { "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" },
+            { ".-   ", "-... ", "-.-. ", "-..  ", ".    ", "..-. ", "--.  ", ".... ", "..   ", ".--- ", "-.-  ", ".-.. ", "--   ", "-.   ", "---  ", ".--. ", "--.- ", ".-.  ", "...  ", "-    ", "..-  ", "...- ", ".--  ", "-..- ", "-.-- ", "--.. ", "-----", ".----", "..---", "...--", "....-", ".....", "-....", "--...", "---..", "----." }};
+            //Use ToCharArray() method for string to copy charecters to Unicode character array
+            //Use foreach loop for character array in which
+            foreach (var c in sos.ToCharArray())
+            {
+                for (var i = 0; i < Dictionary_arr.GetLength(1); i++)
+                {
+                    if (c == char.Parse(Dictionary_arr[0, i]))
+                    {
+                        foreach (var s in Dictionary_arr[1, i].ToCharArray())
```

Comment:
what `s` this mean?

Reply:
Fixed in commit 592bca63f86e969b28bb074d314dd91abb86c457